src/api/feed-discovery/test/router.test.js

```diff
@@ -93,7 +93,7 @@ describe('POST /', () => {
     });
 
     it('should call next() if there is any discovered feed url', (done) => {
-      const validUrl = 'https://LinkWithFeedUrls.com';
+      const validUrl = 'https://LinkWithFeedURLs.com';
```

Comment:
URLs should *always* be lowercase.  You'll drive yourself crazy with debugging on systems that are case sensitive vs. not, otherwise. 

```js
const validUrl = 'https://linkwithfeedurls.com';
```

Reply:
Ah, i see. I misunderstood your point. Will update shortly.